pkgs/servers/misc/navidrome/default.nix

```diff
@@ -1,50 +1,61 @@
-{ lib, stdenv, fetchurl, ffmpeg, ffmpegSupport ? true, makeWrapper, nixosTests }:
+{ callPackage, buildGoModule, fetchFromGitHub, lib, pkg-config, stdenv, ffmpeg, taglib, zlib, ffmpegSupport ? true, makeWrapper, nixosTests }:
 
 with lib;
```

Comment:
This was the lib I was proposing to get rid off.

Reply:
Oops, I misunderstood. Removed the `with lib;` line.